mtags/src/main/scala/scala/meta/internal/pc/CompletionProvider.scala

```diff
@@ -211,10 +211,8 @@ class CompletionProvider(
               // The ideal solution would be to not use the `($0)` snippet when the commit character `(` is used,
               // however language servers can't distinguish what commit character is used. It works as
               // expected in IntelliJ.
-            } else if (r.sym.isError) {
-              () // No commit character
             } else {
-              item.setCommitCharacters(List(".").asJava)
```

Comment:
The whole pattern match can be removed, we can add a comment instead

```scala
// NOTE: We intentionally don't set the commit character because there are valid scenarios where
// the user wants to type a dot '.' character without selecting a completion item.
```